feature/profile/build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+plugins {
+    alias(libs.plugins.mifospay.android.feature)
+    alias(libs.plugins.mifospay.android.library.compose)
+}
+
+android {
+    namespace = "org.mifospay.profile"
+}
+
+dependencies {
+    implementation(projects.core.data)
+    implementation(libs.squareup.okhttp)
```

Comment:
why do you need okhttp here?

Reply:
We need it for ResponseBody and inputStream : private fun convertResponseToBitmap(responseBody: ResponseBody?): Bitmap? {
        return try {
            responseBody?.byteStream()?.use { inputStream ->
                BitmapFactory.decodeStream(inputStream)
            }